src/Raven.Server/Dashboard/ThreadsInfo.cs

```diff
@@ -14,7 +14,7 @@ public class ThreadsInfo : AbstractDashboardNotification
 
         public double CpuUsage { get; set; }
 
-        public long ActiveCores { get; set; }
+        public double ActiveCores { get; set; }
```

Comment:
There is no meaning from our perspective to `0.35` of a core. The extension script needs to do the transaction, not us

Reply:
https://github.com/ravendb/ravendb/pull/7960#discussion_r232563223